pyccl/emulators/ee2_nonlinear_pk.py

```diff
@@ -0,0 +1,59 @@
+__all__ = ("EuclidEmulator2Nonlinear",)
+
+import numpy as np
+
+from .. import Pk2D
+from .. import CCLError
+from . import EmulatorPk
+
+
+class EuclidEmulator2Nonlinear(EmulatorPk):
```

Comment:
It seems we're not testing this one at all. Is there an easy benchmark you could code up?

Reply:
Nope! I will figure something out